### DISCUSSION TITLE
Grain Size calculation

### DISCUSSION THREAD

**Discussion by syedauc** [mtex-toolbox/mtex#765] [Ask Anything]
Hi All,

I use equivalent radius (diameter) as grain size calculation. My question is when we consider CSL grain boundaries how should I calculate grainsize? To my understanding CSL (like sigma 3, twinning) is considered inside a grain not as boundary. Is there anyway I can exclude the CSL 3 from grain size calculation?

**Comment by kilir** [comment]
Hi,
usually, one would merge the grains along twin boundaries before saying something about grain size. Have look here: https://mtex-toolbox.github.io/GrainMerge.html
Cheers,
Rüdiger

**Comment by syedauc** [comment]
Hi  Rüdiger ,
Thanks a lot
